1.5/Source/PawnStorages/PawnStoragesModule-VEF/Init.cs

```diff
@@ -0,0 +1,13 @@
+﻿using PawnStorages.Farm.Comps;
+using Verse;
+
+namespace PawnStorages.VEF;
+
+[StaticConstructorOnStartup]
```

Comment:
Might it be a bit more useful/extensible to instead have the core mod grab any `IExtraProductionHandler` implementations, construct them, then call an `Initialize` on em? Potentially from a `GameComponent` or similar. The currentl implementation of `ProductionHandlerExtraAnimalProduct` looks like it could all just be static anyway, but I can maybe envisage something requiring some state?

Or if not grabbing the implementations of `IExtraProductionHandler`, a `ExtraProductionHandlerDef`?

This way is fine and works, and the above wouldn't break anything if introduced to replace this, so maybe a future todo if you agree with the approach?